src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -142,24 +142,32 @@ define([
                 this.$menu.find('.action').first().focus();
             },
 
+
+            focusLastItem: function() {
+                this.$menu.find('.action.last').focus();
+            },
+
             handlerIsAction: function(key, $el) {
-                if (key === constants.keyCodes.up) {
+                if (key === constants.keyCodes.up || key === constants.keyCodes.left) {
```

Comment:
This is just a suggestion.  Have you thought about using underscore's [contain](http://underscorejs.org/#contains)? e.g. `_([constants.keyCodes.up, constants.keyCodes.left]).contains(key)`.  Maybe even creating member variables and using them::

```
previewKeys = [constants.keyCodes.up, constants.keyCodes.left];
...
if (_(this.previousKeys).contains(key) {...}
```

It may be a bit overkill for two keys though!